README.md

```diff
@@ -29,7 +40,8 @@ These are the main folders within the repository:
     * `Elastic.Apm`: The core project targeting .NET Standard 2.0. It contains the [Agent API](/docs/public-api.asciidoc), the infrastructure to report data to the APM Server, the logging infrastructure, and auto-instrumentation for things that are part of .NET Standard 2.0.
     * `Elastic.Apm.AspNetCore`: Auto-instrumentation for ASP.NET Core.
     * `Elastic.Apm.EntityFrameworkCore`: Auto-instrumentation for EntityFramework Core.
-    * `Elastic.Apm.NetCoreAll`: A convenient project that references all other projects from the `src` folder. It contains an ASP.NET Core middleware extension that enables the agent and every other component with a single line of code. In a typical ASP.NET Core application (e.g. apps referencing [Microsoft.AspNetCore.All](https://www.nuget.org/packages/Microsoft.AspNetCore.All)) that uses EF Core the `Elastic.Apm.All` can be referenced.
+    * `Elastic.Apm.NetCoreAll`: A convenient project that references all other .NET Core related projects from the `src` folder. It contains an ASP.NET Core middleware extension that enables the agent and every other component with a single line of code. In a typical ASP.NET Core application (e.g. apps referencing [Microsoft.AspNetCore.All](https://www.nuget.org/packages/Microsoft.AspNetCore.All)) that uses EF Core the `Elastic.Apm.NetCoreAll` can be referenced.
```

Comment:
Did you mean "A convenience project that ..."?

Reply:
I looked this up, this is very close 🧐 - I changed it for now, probably "convenience" is ok - but I wanna know which is good - by googling it, in this specific case I'm unable to decide; maybe both? @bmorelli25 ? 🙂

Reply:
This is a fun one. Either could work in this situation, although I lean towards convenient as an adjective here.

Reply:
👍 thanks, ok, let's go with `convenient`.